lib/index.js

```diff
@@ -87,5 +87,11 @@ function renderPage(page) {
 function renderLoad(settings) {
   if (!settings.load) return '';
 
+  if (typeof settings.load !== 'boolean') {
+    // eslint-disable-next-line no-restricted-globals
+    var loadOptions = JSON.stringify(settings.load);
+    return 'analytics.load("' + settings.apiKey + '", ' + loadOptions + ');';
+  }
```

Comment:
In order to be fully compliant with the typescript definitions added on #55 it'd be great if we could change the type of `load?: boolean` to something like 

```
type LoadOptions = boolean | { ... }

// ...
load?: LoadOptions
```

Reply:
Added `LoadOptions`. Here is my IDE suggesting the `All` member. I added this one explicitly as it's a known property but can be omitted if preferred. 

![image](https://user-images.githubusercontent.com/10183584/124370180-ba709b80-dcb7-11eb-83b7-be7d5889e532.png)